interface/views.py

```diff
@@ -160,7 +160,7 @@ def done(request, pk):
     assert submission.verify_jwt(request.GET.get('token'))
 
     stdout = utils.decode(options['stdout'])
-    stderr = utils.decode(options['stderr'])
+    stderr = utils.decode(options['stderr'], '')
```

Comment:
You sure this works? [`decode`](https://github.com/vmck/acs-interface/blob/3683329573c9a063608f0675f5d3277449bd5dc0/interface/utils.py#L36) only accepts one argument.

Reply:
Removed the stderr from the ```views``` and ```templates```